### PR TITLE
feat(plasma-ui, plasma-hope, plasma-core): Add package version and info to global variable

### PR DESCRIPTION
**Pull request by neretin-trike**
Были созданы два babel плагина:

* babel-plugin-import-replacer - плагин предназначен для простой замены источников импорта. Подробное описание в [документации](https://github.com/salute-developers/plasma/tree/neretinaa/add-global-version/scripts/babel-plugin-import-replacer)
* babel-plugin-collect-package-info - плагин предназначен для интеграции и вызова метода `collectPackageInfo` во всех файлах, которые реэкспортируются в index.ts на уровне компонент. Подробное описание в [документации](https://github.com/salute-developers/plasma/tree/neretinaa/add-global-version/scripts/babel-plugin-collect-package-info)

Осуществлен рефакторинг ре-экспортов в библиотеках `plasma-ui`, `plasma-core` и `plasma-hope`. Это нужно было для того, чтобы созданный `babel-plugin-collect-package-info` мог корректно считать пути ре-экспортов и записать функцию `collectPackageInfo` в файлы.

Так же реализован метод `collectPackageInfo`, который добавляет глобальную переменную `__PLASMA_INFO__ `. В ней накапливается информация об используемой версии пакета и компонентах, которые были импортированы в формате "<имякомпонента>/<имяфайла>".

Пример получившейся структуры в случае импорта компонента `Button` из библиотеки `@salutejs/plasma-ui`:

```ts
__PLASMA_INFO__: {
    '@salutejs/plasma-core': {
        version: '"1.100.0',
        entities: [
          "Spinner/Spinner",
          "Button/Button",
          "Button/createButton".
          "Button/Button.mixins"
          "Button/Button.props"
        ]
    },
    '@salutejs/plasma-ui': {
        version: '"1.180.0',
        entities: [
          "Button/Button"
        ]
    },
};
```

:baby_chick: Download canary assets:

[colors--canary.460.4665590153.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.460.4665590153.xml)  
[color_sberHealth_ios-swift--canary.460.4665590153.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.460.4665590153.swift)  
[color_sberHealth_kotlin--canary.460.4665590153.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.460.4665590153.kt)  
[color_sberHealth_react-native--canary.460.4665590153.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.460.4665590153.ts)  
[color_sbermarket_business_ios-swift--canary.460.4665590153.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.460.4665590153.swift)  
[color_sbermarket_business_kotlin--canary.460.4665590153.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.460.4665590153.kt)  
[color_sbermarket_business_react-native--canary.460.4665590153.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.460.4665590153.ts)  
[color_sbermarket_ios-swift--canary.460.4665590153.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.460.4665590153.swift)  
[color_sbermarket_kotlin--canary.460.4665590153.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.460.4665590153.kt)  
[color_sbermarket_metro_ios-swift--canary.460.4665590153.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.460.4665590153.swift)  
[color_sbermarket_metro_kotlin--canary.460.4665590153.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.460.4665590153.kt)  
[color_sbermarket_metro_react-native--canary.460.4665590153.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.460.4665590153.ts)  
[color_sbermarket_react-native--canary.460.4665590153.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.460.4665590153.ts)  
[color_sbermarket_selgros_ios-swift--canary.460.4665590153.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.460.4665590153.swift)  
[color_sbermarket_selgros_kotlin--canary.460.4665590153.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.460.4665590153.kt)  
[color_sbermarket_selgros_react-native--canary.460.4665590153.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.460.4665590153.ts)  
[color_sbermarket_vinlab_ios-swift--canary.460.4665590153.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_vinlab_ios-swift--canary.460.4665590153.swift)  
[color_sbermarket_vinlab_kotlin--canary.460.4665590153.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_vinlab_kotlin--canary.460.4665590153.kt)  
[color_sbermarket_vinlab_react-native--canary.460.4665590153.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_vinlab_react-native--canary.460.4665590153.ts)  
[color_sberprime_ios-swift--canary.460.4665590153.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.460.4665590153.swift)  
[color_sberprime_kotlin--canary.460.4665590153.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.460.4665590153.kt)  
[color_sberprime_react-native--canary.460.4665590153.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.460.4665590153.ts)  
[PlasmaTokensColor--canary.460.4665590153.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.460.4665590153.swift)  
[shadow_sbermarket_react-native--canary.460.4665590153.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.460.4665590153.ts)  
[typo_mage_ios-swift--canary.460.4665590153.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.460.4665590153.swift)  
[typo_mage_kotlin--canary.460.4665590153.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.460.4665590153.kt)  
[typo_mage_react-native--canary.460.4665590153.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.460.4665590153.ts)  
[typo_plasma_ios-swift--canary.460.4665590153.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.460.4665590153.swift)  
[typo_plasma_kotlin--canary.460.4665590153.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.460.4665590153.kt)  
[typo_plasma_react-native--canary.460.4665590153.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.460.4665590153.ts)  
[typo_ruler_ios-swift--canary.460.4665590153.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.460.4665590153.swift)  
[typo_ruler_kotlin--canary.460.4665590153.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.460.4665590153.kt)  
[typo_ruler_react-native--canary.460.4665590153.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.460.4665590153.ts)  
[typo_sage_ios-swift--canary.460.4665590153.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.460.4665590153.swift)  
[typo_sage_kotlin--canary.460.4665590153.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.460.4665590153.kt)  
[typo_sage_react-native--canary.460.4665590153.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.460.4665590153.ts)  
[typo_sbermarket_ios-swift--canary.460.4665590153.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.460.4665590153.swift)  
[typo_sbermarket_kotlin--canary.460.4665590153.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.460.4665590153.kt)  
[typo_sbermarket_react-native--canary.460.4665590153.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.460.4665590153.ts)  
[typo_soulmate_ios-swift--canary.460.4665590153.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.460.4665590153.swift)  
[typo_soulmate_kotlin--canary.460.4665590153.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.460.4665590153.kt)  
[typo_soulmate_react-native--canary.460.4665590153.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.460.4665590153.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.174.0-canary.460.4665590153.0
  npm install @salutejs/plasma-core@1.108.0-canary.460.4665590153.0
  npm install @salutejs/plasma-hope@0.32.0-canary.460.4665590153.0
  npm install @salutejs/plasma-icons@1.137.0-canary.460.4665590153.0
  npm install @salutejs/plasma-temple@1.150.0-canary.460.4665590153.0
  npm install @salutejs/plasma-ui@1.182.0-canary.460.4665590153.0
  npm install @salutejs/plasma-web@1.199.0-canary.460.4665590153.0
  npm install @salutejs/plasma-sb-utils@0.106.0-canary.460.4665590153.0
  # or 
  yarn add @salutejs/plasma-b2c@1.174.0-canary.460.4665590153.0
  yarn add @salutejs/plasma-core@1.108.0-canary.460.4665590153.0
  yarn add @salutejs/plasma-hope@0.32.0-canary.460.4665590153.0
  yarn add @salutejs/plasma-icons@1.137.0-canary.460.4665590153.0
  yarn add @salutejs/plasma-temple@1.150.0-canary.460.4665590153.0
  yarn add @salutejs/plasma-ui@1.182.0-canary.460.4665590153.0
  yarn add @salutejs/plasma-web@1.199.0-canary.460.4665590153.0
  yarn add @salutejs/plasma-sb-utils@0.106.0-canary.460.4665590153.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-460/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-460/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-460/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-460/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-460/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-460/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-460/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-460/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-460/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-460/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-460/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-460/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-460/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-460/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-460/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-460/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-460/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-460/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

File: packages/plasma-hope/src/components/Select/index.ts [hunk 1]
```diff
@@ -2,4 +2,4 @@ export { Select } from './Select';
 export type { SelectProps } from './Select';
 export type { SelectRefElement } from './SelectButton';
 
-export { SelectGroup } from './views/b2c/SelectGroup';
```
Comment (Yakutoc) [inline packages/plasma-hope/src/components/Select/index.ts:5]: @neretin-trike Напомни пожалуйста, почему у нас пути изменились?

Comment (neretin-trike) [reply packages/plasma-hope/src/components/Select/index.ts:5]: я там выше в самой прке описал. Поскольку этот файл лежит в index.ts, т.е. пользователи могут его заюзать, то надо чтобы метод `collectPackageInfo` туда тоже залетел. Но это не будет, т.к. SelectGroup лежит глубоко, а плагин смотрит только верхнеуровнево

**Review by Yakutoc — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-460/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-460/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-460/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-460/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-460/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-460/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-460/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-460/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-460/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-460/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-460/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-460/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-460/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-460/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-460/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

File: scripts/babel-plugin-import-replacer/index.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        ImportDeclaration(path, state) {
+            const { alias } = state.opts;
+
+            if (!alias) {
+                return;
+            }
+
+            const aliases = Object.entries(alias);
+
+            aliases.forEach(([key, value]) => {
```
Comment (Yeti-or) [inline scripts/babel-plugin-import-replacer/index.js:12]: этот плагин тоже не описан в доке, что он делает с алиасами? и зачем?

**Review by Yeti-or — COMMENTED** [review]
Пока выглядит read-only

File: packages/plasma-core/src/collectPackageInfo.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        return;
+    }
+
+    const { name, version } = _package;
+
+    const entities = window.__PLASMA_INFO__?.[name]?.entities || [];
+    entities.push(entityName);
+
+    window.__PLASMA_INFO__ = {
+        ...window.__PLASMA_INFO__,
```
[the remark this answers is not in the document]
Comment (Yakutoc) [reply packages/plasma-core/src/collectPackageInfo.ts:27]: @Yeti-or Ты к тому что spread делает копию объекта? Или

[the remark this answers is not in the document]
Comment (Yeti-or) [reply packages/plasma-core/src/collectPackageInfo.ts:27]: Делает копию объекта, 20 сущностей 20 раз все копируем

[the remark this answers is not in the document]
Comment (neretin-trike) [reply packages/plasma-core/src/collectPackageInfo.ts:27]: да, я думаю вариант Васи лучше, спасиб 👌

File: scripts/babel-plugin-collect-package-info/README.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
+# Babel плагин babel-plugin-collect-package-info
+
+Данный плагин предназначен для интеграции и вызова метода `collectPackageInfo` во всех файлах, которые ре-экспортируются в `index.ts` на уровне компонент.
```
[the remark this answers is not in the document]
Comment (neretin-trike) [reply scripts/babel-plugin-collect-package-info/README.md:3]: добавлю 👍

File: scripts/babel-plugin-collect-package-info/index.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const paths = require('path');
+const glob = require('fast-glob');
+
+const getExportPathsFromIndex = require('./getExportPathsFromIndex');
```
[the remark this answers is not in the document]
Comment (neretin-trike) [reply scripts/babel-plugin-collect-package-info/index.js:5]: ну я хотел, чтобы main файл плагина был чистеньким, поэтому все методы разбил на файлы

File: scripts/babel-plugin-import-replacer/index.js (hunk 2 above)
Comment (neretin-trike) [reply scripts/babel-plugin-import-replacer/index.js:12]: на самом деле описал, и в шапке, и в [доке](https://github.com/salute-developers/plasma/tree/neretinaa/add-global-version/scripts/babel-plugin-import-replacer) :)

File: scripts/babel-plugin-collect-package-info/getExportPathsFromIndex.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    indexFiles.reduce((acc, src) => {
+        const code = fs.readFileSync(src, 'utf-8');
+        const ast = parser.parse(code, parseOptions);
+        const folderName = paths.basename(paths.dirname(src));
+
+        traverse(ast, {
+            ExportNamedDeclaration(path) {
+                const specifiers = path.get('specifiers');
+
+                if (!specifiers.length) {
```
[the remark this answers is not in the document]
Comment (neretin-trike) [reply scripts/babel-plugin-collect-package-info/getExportPathsFromIndex.js:24]: для таких кейсов, когда в `index.ts` экспортируемая сущность не имеет путей ре-экспорта / импорта, например:

```
export const Slider = SliderCore;
```

Но в данном случае в `Slider` метод всё-равно попадёт, т.к.  выше есть код 

```
import { SliderCore } from './Slider';
```

File: scripts/babel-plugin-collect-package-info/index.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (Yeti-or) [reply scripts/babel-plugin-collect-package-info/index.js:5]: я имел ввиду что есть главный экспорт, и там же внутри бабеля происходит парсинг, 
а еще ты сам вызываешь parse ast все дела внутри `getExportPathsFromIndex` хотелось понять зачем

File: scripts/babel-plugin-collect-package-info/getExportPathsFromIndex.js (hunk 6 above)
[the remark this answers is not in the document]
Comment (Yeti-or) [reply scripts/babel-plugin-collect-package-info/getExportPathsFromIndex.js:24]: объясни мне еще раз пожулайста я тупой

[the remark this answers is not in the document]
Comment (neretin-trike) [reply scripts/babel-plugin-collect-package-info/getExportPathsFromIndex.js:24]: наверное я не очень понятно объяснил прост. Есть, например файл /Slider/index.ts, в нём лежит такая структура:

```
import { SliderCore } from './Slider';
export { SliderCore } from './Slider';
export { ThumbBase } from './ThumbBase';
export type { SliderSettings, SliderProps, SingleSliderProps, DoubleSliderProps } from './types';
export const Slider = SliderCore;
```

Колбэк `ExportNamedDeclaration` бежит по всем именованным импортам / экспортам и ищет в них `specifiers`, чтобы определить, что конкретно этот узел имеет путь для импорта / экспорта (`... from 'path')`, затем находит и добавляет его в мою мапу. В данном примере это будет `Slider`, `ThumbBase` и `types`.

Но когда доходит до последней строчки, то не обнаруживает там путей для импорта / экспорта, прокидывает path и дальше по коду ломается в методе `accumulateExportPaths`,  т.к. пытается найти `source`, которого нет

File: scripts/babel-plugin-collect-package-info/index.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (neretin-trike) [reply scripts/babel-plugin-collect-package-info/index.js:5]: а, понял о чём ты. Это нужно для того, чтобы сначала сформировать мапу с компонентами/сущностями **только** в файле `index.ts`.

И затем, когда уже в главном экспорте в колбэке `Program` будет строиться ast, то я буду вставлять метод `collectPackageInfo` только в нужные мне файлы на основании мапы, которую я сделал в `getExportPathsFromIndex `. 

А заставить бежать по файлам в нужном мне порядке я его не могу. По скорости, кстати, никакой ощутимой просадки нет, он всё также быстро работает

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-460/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-460/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-460/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-460/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-460/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-460/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-460/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

File: scripts/babel-plugin-collect-package-info/getExportPathsFromIndex.js (hunk 6 above)
[the remark this answers is not in the document]
Comment (Yeti-or) [reply scripts/babel-plugin-collect-package-info/getExportPathsFromIndex.js:24]: понял, то есть нам не важно в данном случае что `export const Slider = SliderCore;` или как он вообще экспортируется, а важно что этот файл вызвали так?

File: scripts/babel-plugin-collect-package-info/index.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (Yeti-or) [reply scripts/babel-plugin-collect-package-info/index.js:5]: понял спасибо 👍

File: scripts/babel-plugin-import-replacer/README.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
+# Babel плагин babel-plugin-import-replacer
+
+Данный плагин предназначен для простой замены источников импорта.
+
+## Описание
+
+С помощью плагина происходит "плоская" (т.е. без учёта вложенности) подмена импортов на основе синонимов (`alias`). Параметр `alias` принимает в себя массив объектов в формате "<имя__синонима>:<целевой__путь>".
+
+## Пример использования
```
Comment (Yeti-or) [inline scripts/babel-plugin-import-replacer/README.md:9]: 👍

**Review by Yeti-or — APPROVED** [review]

File: scripts/babel-plugin-collect-package-info/getExportPathsFromIndex.js (hunk 6 above)
[the remark this answers is not in the document]
Comment (neretin-trike) [reply scripts/babel-plugin-collect-package-info/getExportPathsFromIndex.js:24]: скорее нам вообще не важно его существование `Slider`, т.к. `SliderCore` уже импортируется

**Pull request merged**